src/app/static/src/components/adr/SelectRelease.vue

```diff
@@ -125,7 +123,7 @@
                 return (this.choiceADR === "useLatest") || !!this.releaseId;
             },
             releaseOptions() {
-                return this.releases.map((d) => ({
+                return (this.releases as Release[]).map((d) => ({
```

Comment:
it's really odd that you've had to cast this everywhere, suggests the type qualifiers on line 102 aren't working as expected?